src/Swashbuckle.AspNetCore.Swagger/SwaggerMiddleware.cs

```diff
@@ -72,16 +88,37 @@ public async Task Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvid
             }
         }
 
-        private bool RequestingSwaggerDocument(HttpRequest request, out string documentName)
+        private bool RequestingSwaggerDocument(HttpRequest request, out string documentName, out string extension)
         {
             documentName = null;
+            extension = null;
             if (request.Method != "GET") return false;
 
             var routeValues = new RouteValueDictionary();
-            if (!_requestMatcher.TryMatch(request.Path, routeValues) || !routeValues.ContainsKey("documentName")) return false;
+            if (_requestMatcher.TryMatch(request.Path, routeValues))
+            {
+#if !NETSTANDARD
+                if (!_templateBinder.TryProcessConstraints(request.HttpContext, routeValues, out _, out _))
```

Comment:
We should null guard this, as it's technically possible for someone using the TFM with this support to use the constructor that doesn't provide a value, and then it'll `NullReferenceException`.

Reply:
Yes you're right! I just addressed this in f30392a4f97d694a2e53fd53bb462d29202a8a7d.

I don't know how to code without nullable reference types anymore. 😅 (This would have caught this error by the way.)

Reply:
We have an issue to track that it turns out: #2374